server/database/sql/create_views.sql

```diff
@@ -0,0 +1,7 @@
+create view dashboard_words as
```

Comment:
@ChrisShep98 these are views. you have to specify a known schema type to be returned from all functions, so views are a way to store these joins in the database so that we can refer to them in the function. you can see in the two get_dashboard_words/sentences functions the return types refer to these views, so that we don't have to list all the fields we plan to return from the tables

while typing this explanation I'm realizing it's kind of like storing types in a separate file in typescript so that you can refer to them by name in your application. the difference tho is that a view is also a table, so you can query these tables the same as any. It's just a way to store a pointer to a join that you often use